src/app/CommandHandlerImpl.h

```diff
@@ -294,7 +294,13 @@ class CommandHandlerImpl : public CommandHandler
         {
             return CHIP_NO_ERROR;
         }
-        ReturnErrorOnFailure(RollbackResponse());
+        // The error value of RollbackResponse is not important if it fails, we prioritize
+        // conveying the error generated by addResponseFunction to the
+        // caller.
```

Comment:
nit, you can have the next line moved up to here it seems odd with the alignment of the comment above to have a 3rd line.

Sorry it is hard to see the alignment stuff when writing the suggestion but it is easier to see if the review after you apply the suggestion

```suggestion
        // conveying the error generated by addResponseFunction to the caller.
```

Reply:
Done.